src/Symfony/Component/PropertyAccess/StringUtil.php

```diff
@@ -203,7 +203,7 @@ public static function singularify($plural)
         }
 
         // Convert teeth to tooth, feet to foot
-        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3 && 'feedback' !== $plural) {
+        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3 && !preg_match('{feedback|Feedback|career|Career}', $plural)) {
```

Comment:
Yes, the main entity in the form is ```Career```. I could create a fork from ```symfony-standard``` in order to try to reproduce this issue if required.